### ISSUE TITLE
Response is double-quoted

### ISSUE THREAD

**Issue by jayadevanm** [graphql-python/gql#377]
The response in given in the example 
`$ python basic_example.py.                                                     
{'continents': [{'code': 'AF', 'name': 'Africa'}, {'code': 'AN', 'name': 'Antarctica'}, {'code': 'AS', 'name': 'Asia'}, {'code': 'EU', 'name': 'Europe'}, {'code': 'NA', 'name': 'North America'}, {'code': 'OC', 'name': 'Oceania'}, {'code': 'SA', 'name': 'South America'}]}`

If I try the same from, say Postman, the response is double-quoted.

`{
    "data": {
        "continents": [
            {
                "code": "AF",
                "name": "Africa"
            },..`
In short, the response from gql client is not JSON and I am working around that by using json.dumps(). Is it possible to get double-quoted response?

**Comment by leszekhanusz** [comment]
The response from the gql execute method is not text, it is a standard Python dict.

So to generate JSON from it, using `json.dumps()` is not a workaround, it is the normal way of doing it.
This is done for example when you use [gql-cli](https://gql.readthedocs.io/en/stable/gql-cli/intro.html#simple-query-using-https), you'll get double-quotes there as we use [json.dumps in the code](https://github.com/graphql-python/gql/blob/f28670da7656d195487cbf6417ac16571a4951f2/gql/cli.py#L434)

**Comment by jayadevanm** [comment]
Okay, thanks a lot. Great tool, by the way.
